scripts/utils/republishUtils.js

```diff
@@ -1,27 +1,20 @@
-const esUtils = require('./esUtils');
+import esUtils from './esUtils.js';
 
 // Do some env setup to simulate using pm2 as an env import (amazing hack!)
 process.env = esUtils.config;
 
-require('babel-polyfill');
-require('@babel/register')({
-  presets: ['@babel/env'],
-  plugins: [
-    '@babel/plugin-proposal-optional-chaining',
-    '@babel/plugin-proposal-object-rest-spread',
-  ],
-});
```

Comment:
Note: I removed this babel import entirely as this module isn't compatible with ESM, and the plugins are for ES6 features we already have access to natively in Node 24 now.